codespell_lib/tests/test_basic.py

```diff
@@ -233,6 +233,37 @@ def test_ignore_word_list(tmpdir, capsys):
     assert cs.main('-Labandonned,someword', '-Labilty', d) == 1
 
 
+@pytest.mark.parametrize('content, expected_error_count', [
```

Comment:
You haven't tested where the typo doesn't exist in the line at all, but is still being erroneously ignored!